elixir/apps/api/test/api/controllers/actor_controller_test.exs

```diff
@@ -0,0 +1,213 @@
+defmodule API.ActorControllerTest do
+  alias API.Gateway.Views.Actor
+  alias API.Gateway.Views.Actor
+  use API.ConnCase, async: true
+  alias Domain.Actors.Actor
+
+  setup do
+    account = Fixtures.Accounts.create_account()
+    actor = Fixtures.Actors.create_actor(type: :api_client, account: account)
+
+    %{
+      account: account,
+      actor: actor
+    }
+  end
+
+  describe "index" do
```

Comment:
We should add `/arity` in the of describe blocks, it makes it easier to find tests by grep'ing and it just a general convention for Elixir:
```suggestion
  describe "index/1" do
```

Reply:
Sounds good 👍